cognite/client/_api/relationships.py

```diff
@@ -0,0 +1,172 @@
+from typing import *
+
+from cognite.client import utils
+from cognite.client._api_client import APIClient
+from cognite.client.data_classes import Relationship, RelationshipList
+from cognite.client.exceptions import CogniteAPIError
+
+
+class RelationshipsAPI(APIClient):
+    _RESOURCE_PATH = "/relationships"
+    _LIST_CLASS = RelationshipList
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._CREATE_LIMIT = 1000
+
+    def __call__(
+        self, chunk_size: int = None, limit: int = None
+    ) -> Generator[Union[Relationship, RelationshipList], None, None]:
+        """Iterate over relationships
+
+        Fetches relationships as they are iterated over, so you keep a limited number of relationships in memory.
+
+        Args:
+            chunk_size (int, optional): Number of relationships to return in each chunk. Defaults to yielding one relationship a time.
+            limit (int, optional): Maximum number of relationships to return. Defaults to 100. Set to -1, float("inf") or None
+                to return all items.
+
+        Yields:
+            Union[Relationship, RelationshipList]: yields Relationship one by one if chunk is not specified, else RelationshipList objects.
+        """
+        return self._list_generator(method="POST", chunk_size=chunk_size, limit=limit)
+
+    def __iter__(self) -> Generator[Relationship, None, None]:
+        """Iterate over relationships
+
+        Fetches relationships as they are iterated over, so you keep a limited number of relationships in memory.
+
+        Yields:
+            Relationship: yields Relationships one by one.
+        """
+        return self.__call__()
+
+    def retrieve(self, external_id: str) -> Optional[Relationship]:
+        """Retrieve a single relationship by external id.
+
+        Args:
+            external_id (str): External ID
+
+        Returns:
+            Optional[Relationship]: Requested relationship or None if it does not exist.
+
+        Examples:
+
+            Get relationship by external id::
+
+                >>> from cognite.client import CogniteClient
```

Comment:
Examples should import CogniteClient from experimental namespace, and be added to test_docstring_examples.py